sidebars.json

```diff
@@ -5,6 +5,7 @@
       "WebEditorBasics/WebEditorInterface",
       "WebEditorBasics/MinimumRequirements",
       "WebEditorBasics/ManagingYourExperiences",
+	  "WebEditorBasics/SharingExperiences",
```

Comment:
Small thing: please make it so your text editor converts tabs to spaces and uses 2 spaces for indentation.

Reply:
done!

Reply:
hopefully it looks properly formatted now